interface.js

```diff
@@ -0,0 +1,84 @@
+const articles = []
+
+window.onload = function() {
+  getHeadlines()
+}
+
+window.addEventListener("hashchange", function(){
+  let id = window.location.hash.split("#")[1];
+  displayArticle(articles[id]);
+})
+
+function displayArticle(article) {
+  var modal = document.getElementById("myModal");
+  modal.style.display = "block";
+  // getSummary(article, "modal-text")
+  var body = `${article.webTitle} (click for more)`
+  document.getElementById("modal-text").innerHTML = body.link(article.webUrl)
+  document.getElementsByClassName("close")[0].onclick = function() {
+    modal.style.display = "none";
+    history.replaceState(null, null, ' ');
+  }
```

Comment:
Wonder if perhaps any of this could be extracted to a separate function? quite chunky